airbyte-integrations/connectors/source-mssql/build.gradle

```diff
@@ -34,3 +34,5 @@ dependencies {
     implementation files(project(':airbyte-integrations:bases:base-java').airbyteDocker.outputs)
     // integrationTestImplementation files(project(':airbyte-integrations:bases:base-java').airbyteDocker.outputs)
 }
+
+compileJava.dependsOn(':airbyte-integrations:connectors:source-mysql:build')
```

Comment:
why is that?

Reply:
It seems like a hack, we should at least have a comment explaining why and how to get rid of it

Reply:
shouldn't be here.